package.json

```
@@ -427,6 +427,16 @@
 					"type": "boolean",
 					"default": false,
 					"description": "%asciidoc.useWorkspaceRoot.desc%"
+				},
+				"asciidoc.antora": {
```

I think we should create a category for Antora:

```suggestion
				"asciidoc.antora.enableAntoraSupport": {
```